src/index.js

```diff
@@ -7,12 +7,14 @@ const JwtVerifyError = require('./jwtverifyerror')
 const JwtServiceAuthError = require('./jwtserviceautherror')
 const JwtAuthMiddleware = require('./jwtauthmiddleware')
 const JwtServiceAuth = require('./jwtserviceauth')
+const PubkeysHelper = require('./pubkeyshelper')
 
 module.exports = {
   JwtUtils: {
     encode: jwtEncode,
     decode: jwtDecode
   },
+  PubkeysHelper,
```

Comment:
The K in keys should be capital in pascal casing

Reply:
I'm referring to pubkeys variable here, I have it all over the examples and code base.